pxtlib/webusb.ts

```diff
@@ -197,40 +225,38 @@ namespace pxt.usb {
 
             this.enabled = true;
             this.log("registering webusb events");
-            (navigator as any).usb.addEventListener('disconnect', this.handleUSBDisconnected, false);
-            (navigator as any).usb.addEventListener('connect', this.handleUSBConnected, false);
+            navigator.usb?.addEventListener('disconnect', this.handleUSBDisconnected, false);
+            navigator.usb?.addEventListener('connect', this.handleUSBConnected, false);
         }
 
         disable() {
             if (!this.enabled) return;
 
             this.enabled = false;
             this.log(`unregistering webusb events`);
-            (navigator as any).usb.removeEventListener('disconnect', this.handleUSBDisconnected);
-            (navigator as any).usb.removeEventListener('connect', this.handleUSBConnected);
+            navigator?.usb.removeEventListener('disconnect', this.handleUSBDisconnected);
```

Comment:
`usb` is declared optional in the interface. Do you want to honor that here?
```suggestion
            navigator?.usb?.removeEventListener('disconnect', this.handleUSBDisconnected);
```

Reply:
Whoops, i intended to put them there yeah, usb feature will be disabled overall if `navigator.usb` is unset so this was for clarity but fixing, thanks!